lib/protocol/bep_extensions.go

```diff
@@ -213,6 +213,17 @@ func (f FileInfo) isEquivalent(other FileInfo, ignorePerms bool, ignoreBlocks bo
 	return false
 }
 
+func ModTimeEqual(a, b time.Time, modTimeWindowS int) bool {
+	if a == b {
```

Comment:
This should be `a.Equal(b)` to correctly handle time zones and monotonic counters and whatnot